src/Engine/Objects/PlayerEnums.h

```diff
@@ -235,7 +235,7 @@ inline void SetSkillMastery(PLAYER_SKILL *skill_value, PLAYER_SKILL_MASTERY mast
 }
 
 #pragma warning(push)
-#pragma warning(disable : 4341)
+#pragma warning(disable : 4341) // TODO(captainurist): what is this about?
```

Comment:
https://social.msdn.microsoft.com/Forums/vstudio/en-US/fc45de2d-e516-4162-9ec4-de5fddc6681f/why-is-this-enum-throwing-c4341-warnings?forum=vclanguage

Reply:
Eh, that's msvc spedific. OK thanks.